### ISSUE TITLE
Reference for AccountAllowanceAdjustTransaction no longer exists in Hedera Services

### ISSUE THREAD

**Issue by mehcode** [hashgraph/hedera-sdk#57]
I recommend deleting from here. It's present as deprecated in the Java and Go SDKs (not the JavaScript one). The service itself no longer exists in Hedera Services or Protobufs of any form (not even a deprecated form).

**Comment by jg-swirlds** [comment]
Yep, seems totally reasonable! Just need to work out when we should break things.
